index.html

```diff
@@ -81,19 +89,22 @@ <h2>Tonic</h2>
                     <div>
                         <button class="project-button">See Project</button>
```

Comment:
- [ ] Kindly add button interaction styles (hover, clicked...) as shown in the Figma design:

![image](https://user-images.githubusercontent.com/50721479/194669346-31ff0395-7c5f-44d3-b3b5-5f9a34de86d9.png)